topping_bot/util/image.py

```diff
@@ -329,6 +328,13 @@ def topping_text_sort(x: Topping):
             anchor="ma",
         )
 
+    if name:
+        cookie = Cookie.get(name)
```

Comment:
wondering if we should use exact match instead. 
in case user types in the names strangely in the req file and matches another cookie.

Reply:
I think doing fuzzed match is fine, try to catch as much as possible. However we need to check that we can't find a cookie with the provided name

Reply:
Also I need to update cookie assets, I'll include the head icons that I think would be better suited for imposing over topping sets.